katas/content/teleportation/index.md

```diff
@@ -0,0 +1,33 @@
+# Teleportation
+
+@[section]({
+    "id": "teleportation__overview",
+    "title": "Overview"
+})
+
+Teleportation quantum kata is a series of exercises designed to get you familiar with programming in Q#. It covers the quantum teleportation protocol which allows you to communicate a quantum state using only classical communication and previously shared quantum entanglement.
+
+- Teleportation is described in <a href="https://en.wikipedia.org/wiki/Quantum_teleportation">this Wikipedia article</a>.
```

Comment:
Prefer standard markdown links over inline HTML (here and elsewhere) <https://www.markdownguide.org/basic-syntax/#links>

```suggestion
- Teleportation is described in [this Wikipedia article](https://en.wikipedia.org/wiki/Quantum_teleportation).
```

Reply:
Updated